handbook/business-operations/README.md

```diff
@@ -171,6 +171,18 @@ When a Fleetie, consultant or advisor requests an update to their personnel deta
 - If required, BizOps also makes changes to other core systems (e.g: creating a new email alias in google workspace; updating details in Carta; etc).
 - The change is now actioned, notify the team member and close the issue.
 
+### Updating compensation
+When Fleet [changes compensation for team members](https://fleetdm.com/handbook/company/communications#compensation-changes), the Business Operations team adheres to the following process:
+- On the first Friday of the month, the Head of BizOps posts in the `#help-classified` channel with the list of team members due for compensation review.
+- The Head of BizOps creates a copy of the [Values assessment template](https://docs.google.com/spreadsheets/d/1P5TyRV2v-YN0aR_X8vd8GksKcr3uHfUDdshqpVzamV8/edit?usp=drive_link) and stores it in the team member's [personnel file](https://drive.google.com/drive/folders/1UL7o3BzkTKnpvIS4hm_RtbOilSABo3oG?usp=drive_link), then shares with the manager to perform a values assessment.
+- Once the values assessment is complete, the Head of BizOps prepares benchmarking information in the [¶¶ 💌 Compensation decisions](https://docs.google.com/document/d/1NQ-IjcOTbyFluCWqsFLMfP4SvnopoXDcX0civ-STS5c/edit?usp=sharing)file, and confers with manager or head of department about level of job skill in relation to compensation benchmarking levels.
```

Comment:
Lines 178 - 182 can be pulled out and made a separate responsibility (e.g. Benchmark a current team member's compensation).

Reply:
Can you share a use case where we would benchmark a team member's compensation _without_ already considering a compensation change? I think the two are inherently linked, and therefore don't believe we need to separate.

Reply:
Mike: Think more about breakdown of work we could have teammates do, eg: could have Head of BizOps run benchmarks, but may have BizOps Engineer actually make the changes in HRIS. Also take out any assignment of role, that can be tracked by who is assigned the issue rather than who is written here.

Reply:
Sorry, after re-reading what I had above it's confusing!😵‍💫

I'm thinking we have "Update a team member's compensation" in which,  we link to "Prepare benchmark information" (which also happens with new hires and could contain a note on how to prepare a _current_ team member's benchmarking information). It could also be a separate responsibility (e.g. prepare a current team member's benchmarking information) or if it lives encapsulated in "Update a team member's compensation" all good we can just list the steps on how to do that here. Even if its just a stub: 

```
### Prepare benchmark information
TODO: Document this
```

Which will make it a variable we need to define rather than non-existent.